src/polyfill.web.tsx

```diff
@@ -1 +1,56 @@
-export * from './mock';
\ No newline at end of file
+import * as React from 'react';
+import { View } from 'react-native';
+import { AppearancePreferences, ColorSchemeName, AppearanceListener } from './Appearance.types';
+
+type Mode = 'light' | 'dark';
+
+const isDarkQuery = window.matchMedia('(prefers-color-scheme: dark)');
+const isNoPreferenceQuery = window.matchMedia('(prefers-color-scheme: no-preference)');
+const isLightQuery = window.matchMedia('(prefers-color-scheme: light)');
+const isSupported = isDarkQuery.matches || isNoPreferenceQuery.matches || isLightQuery.matches;
+
+const initialMode: Mode = isSupported
+  ? window.matchMedia('(prefers-color-scheme: dark)').matches
+    ? 'dark'
+    : 'light'
+  : 'light';
```

Comment:
it should default to `'no-preference'` if not supported, see: https://snack.expo.io/Hy-xEWeYB

Reply:
done, can you check?